h2o-assemblies/main/build.gradle

```diff
@@ -49,7 +49,8 @@ for (comp in optionalComponents) {
     
 }
 
-shadowJar {
+shadowJar { 
+  zip64 true
```

Comment:
How come this was not needed before? We have tests that verify importing ORC files

Reply:
it is used already in hadoop assembly. we probably hit limit, when pavel was debugging ORC and include ORC into default main assembly.

Reply:
Yep, I want to make sure that enabling the ORC doesn't bring unwanted dependencies.

What I mean is enabling it is fine but we should also look what dependencies get us over the limit.